packages/web/src/app/doc-pages/about/tutorial/blazor-tutorial/blazor-tutorial.component.html

```diff
@@ -0,0 +1,147 @@
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Build-script setup</div>
+    <p class="e-my-8">Create a script in package.json to build the file you created.</p>
+    <app-code-viewer [tabs]="['JSON']" [fileName]="'package.json'" [activeCode]="codeStep5" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Run the build script</div>
+    <p class="e-my-8">Run the new build script that was created.</p>
+    <app-code-viewer [tabs]="['Bash']" [fileName]="'Terminal'" [activeCode]="codeStep6" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Import built files</div>
+    <p class="e-my-8">
+      Import the stylesheet and script that was built. The js/index.bundle.js script should be added after the
+      framework/blazor.web.js script.
+    </p>
+    <app-code-viewer [tabs]="['HTML']" [fileName]="'App.razor'" [activeCode]="codeStep7" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Start using classes</div>
+    <p class="e-my-8">
+      Choose between multiple
+      <a [routerLink]="['/tools/utility-classes']" class="e-link e-link--inline">brand utilities</a> and
+      <a [routerLink]="['/components']" class="e-link e-link--inline">components</a>. The components have
+      multiple variants and options to fit different needs.
+    </p>
+    <app-code-viewer [tabs]="['HTML']" [fileName]="'Home.razor'" [activeCode]="codeStep8" hideTabs />
+  </li>
+</ol>
+
+<!--ADVANCED TUTORIAL-->
+<h3 class="e-title-small e-mt-64">Add an advanced component</h3>
+<ol class="e-list e-list--numbers e-mt-40">
+  <li>
+    <div class="e-title-xsmall">Install component</div>
+    <p class="e-mt-8">
+      Install the
+      <a [routerLink]="['/components/datepicker']" class="e-link e-link--inline">datepicker component.</a>
+    </p>
+    <p class="e-my-8 e-text-md">
+      <strong
+        >The packages acts like any other external dependency you install and update in your project.
+      </strong>
+    </p>
+    <app-code-viewer [tabs]="['Bash']" [fileName]="'Terminal'" [activeCode]="codeStepAdvanced1" hideTabs />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Add imports</div>
+    <p class="e-my-8">Import the component.</p>
+    <app-code-viewer
+      [tabs]="['Typescript']"
+      [fileName]="'index.js'"
+      [activeCode]="codeStepAdvanced2"
+      hideTabs
+    />
+  </li>
+  <li class="e-mt-24">
+    <div class="e-title-xsmall">Start using advanced components</div>
+    <p class="e-mt-8">
+      Choose between several different advanced component and customize according to your needs by using the
+      available props. The
+      <a [routerLink]="['/components/datepicker/']" fragment="API" class="e-link e-link--inline"
+        >datepicker API</a
+      >
+      lists all the properties and events available for datepicker.
+    </p>
+    <div class="e-alert e-alert--info e-my-16" role="alert">
```

Comment:
```suggestion
    <div class="e-alert e-alert--info e-my-16" role="note">
```

Or remove as it is not super-important in this context